cmd/era/main.go

```diff
@@ -11,21 +13,30 @@ import (
 func main() {
 	host := flag.String("h", "", "host address, required")
 	configFilename := flag.String("c", "", "config file, required")
-	out := flag.String("o", "", "output file, required")
+	outputRoot := flag.String("output-root", "", "output file for root certificate")
+	outputIntermediate := flag.String("output-intermediate", "", "output file for intermediate certificate")
+	outputChain := flag.String("output-chain", "", "output file for certificate chain")
 	skipQuote := flag.Bool("skip-quote", false, "skip quote verification")
 	flag.Parse()
-	if *host == "" || *configFilename == "" || *out == "" {
+
+	var noOutputGiven bool
+	if *outputRoot == "" && *outputIntermediate == "" && *outputChain == "" {
```

Comment:
Should we output the chain as the default?

Reply:
You mean with the -o option? 

Cause right now this version doesn't have it anymore, though for ease of use it might be useful to keep it with a default. Unless we want to be very explicit on what kind of certificate the user is dumping from the product.

Reply:
I find it hard to choose a default here, so my idea was to let the user explicitly decide.